src/AnnouncementBubble/NormalAnnouncementBubble.tsx

```diff
@@ -161,7 +166,7 @@ function formReadReceiptsTooltip(
         textAlign={"left"}
       >
         <div onMouseEnter={onReadReceiptsHover}>
-          <FlexBox>
+          <FlexBox onClick={onReadReceiptsClick}>
```

Comment:
For accessibility purposes, it's best practice to apply `onClick` handlers to truly clickable elements, like `button`s and `a`s. The Dewey `FlexBox` just renders a `div` under the hood.

I'd recommend changing this Dewey `FlexBox` to [a Dewey `Button`](https://clever.github.io/components/#/components/button) if you want to make it clickable! I think `<Button type="linkPlain">` should give you the desired styling.

Reply:
As another potentially simpler alternative, if using a Dewey button will require too many styling overrides, you can also apply `role="button" tabIndex={0}` to the `FlexBox`, which will tell screen readers to treat it as a button.

Reply:
Good catch! I went with the latter solution instead. The tabIndex is a nifty prop as well, something that I discovered just now!